libs/database/src/lib/DBLocalStorage.ts

```diff
@@ -1,10 +1,23 @@
-import type { DBStorage } from './DBStorage'
+import type { DbIndexKey, DbVersionKey } from './DBStorage'
+import { type DBStorage, type StorageType } from './DBStorage'
 
 export class DBLocalStorage implements DBStorage {
   private storage: Storage
+  dbVersionKey: DbVersionKey
+  dbIndexKey: DbIndexKey
 
-  constructor(storage: Storage) {
+  constructor(storage: Storage, storageType: StorageType = 'go') {
```

Comment:
i was thinking `(storage:Storage, dbVersionKey:string, dbIndexKey:string)`, seems like you are doing a lot of work around here to keep these keys in sync, when it should just be supplied during initialization.

Reply:
I honestly don't see a huge issue with this. It's just a switch-case statement and ensures that there is never improper crossing of keys that should be synced

Reply:
i see. then good enough.